src/main/resources/application-dev.yaml

```diff
@@ -0,0 +1,24 @@
+management:
```

Comment:
Don't think we're needing anything in here anymore, so can delete this whole file?

Reply:
done